pack/nodejs/helm.ts

```diff
@@ -65,6 +66,9 @@ export namespace v2 {
                     yaml: [yamlStream],
                     transformations: config.transformations || [],
                 }, { parent: this });
+            } catch (e) {
+                console.error(e.toString());
+                process.exit(1);
```

Comment:
This is pretty surprising. An uncaught exception should already tear the process down. Is this not happening?

Reply:
Let's change this to throw a `pulumi.RunError`.